### PR TITLE
Update `The_Team` Listings

### PR DESCRIPTION
**Pull request by Blooshing**
NEW YEAR NEW STAFF LOG

(>‿◠)✌

(っ＾▿＾)۶🍸🌟🍺٩(˘◡˘ )

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/People/The_Team/Beatmap_Nominators/en.md [hunk 1]
```diff
@@ -194,41 +197,46 @@ The tables listed below indicate the game mode(s) in which each Beatmap Nominato
 
 | Name | Additional languages |
 | :-- | :-- |
+| ::{ flag=ID }:: [-Keitaro](https://osu.ppy.sh/users/3378391) | Indonesian |
```
Comment (aceticke) [inline wiki/People/The_Team/Beatmap_Nominators/en.md:200]:
```suggestion
```

-Keitaro is a Full Beatmap Nominator, apply for all languages

File: wiki/People/The_Team/Staff_Log/2023/en.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+Abbreviations for user groups are used throughout this log:
+
+- **BN:** [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators)
+- **GMT:** [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team)
+- **NAT:** [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team)
+
+## January
+
+- 2023-01-14: Removed [Quenlla](https://osu.ppy.sh/users/4725379) from **NAT** to **osu! Alumni**
```
Comment (aceticke) [inline wiki/People/The_Team/Staff_Log/2023/en.md:11]:
```suggestion
- 2023-01-14: Moved [Quenlla](https://osu.ppy.sh/users/4725379) from **NAT** to **osu! Alumni**
```

File: wiki/People/The_Team/Staff_Log/2023/en.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+#### Additions
+
+- 2023-01-03: Added [Ryax](https://osu.ppy.sh/users/8949769) to **Probationary BN**
+- 2023-01-09: Added [secXcscX](https://osu.ppy.sh/users/13543418) to **BN**
+- 2023-01-09: Added [Sparhten](https://osu.ppy.sh/users/7601720) to **Probationary BN**
+- 2023-01-09: Added [Seto Kousuke](https://osu.ppy.sh/users/2857314) to **Probationary BN**
+- 2023-01-09: Added [Krisom](https://osu.ppy.sh/users/99269) to **Probationary BN**
+- 2023-01-09: Added [Bazz B](https://osu.ppy.sh/users/9063995) to **Probationary BN**
+- 2023-01-09: Added [rosario wknd](https://osu.ppy.sh/users/6341518) to **Probationary BN**
+- 2023-01-09: Added [-Keitaro](https://osu.ppy.sh/users/3378391) to **Probationary BN**
```
Comment (aceticke) [inline wiki/People/The_Team/Staff_Log/2023/en.md:24]:
```suggestion
- 2023-01-09: Added [-Keitaro](https://osu.ppy.sh/users/3378391) to **BN**
```

**Review by aceticke — CHANGES_REQUESTED** [review]

File: wiki/People/The_Team/Staff_Log/2023/en.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+- Go to the #groupfeed channel of Aiess Discord server <https://discord.gg/2XV5dcW>
+- Use the "last updated" marker above to find the relevant message sent by Aiess
+- Add entries for all of the messages after that one
+  - For dates, use the timestamp in Aiess embeds (NOT the date of the Discord message), converted to UTC
+  - If a user was added from one group and removed from another at the same time, it can be logged as a "Move" here. The only time this shouldn't be done is when one of the groups is BN and another is not, since we group BN logging separately in this article (e.g. remove from BN + add to NAT != move from BN to NAT, for logging purposes)
+- Update the "last updated" marker above with the current time and the last message from Aiess
+- Make the same updates to group member lists in their relevant articles
+
+—clayton
+-->
```
Comment (aceticke) [inline wiki/People/The_Team/Staff_Log/2023/en.md:56]:
```suggestion
-->

```
new line to fix error

File: wiki/People/The_Team/Staff_Log/2022/en.md [hunk 5]
```diff
@@ -609,28 +612,28 @@ Abbreviations for user groups are used throughout this log:
 - 2022-12-13: Added [Feerum](https://osu.ppy.sh/users/4815717) to **Probationary BN**
 - 2022-12-14: Added [ArtZone](https://osu.ppy.sh/users/15548034) to **Probationary BN**
 - 2022-12-19: Added [Smoke](https://osu.ppy.sh/users/10726630) to **Probationary BN**
+- 2022-12-22: Added [Rocma](https://osu.ppy.sh/users/566276) to **Probationay BN**
+- 2022-12-22: Added [browiec](https://osu.ppy.sh/users/9426712) to **Probationary BN**
+- 2022-12-24: Added [NexusQI](https://osu.ppy.sh/users/13822800) to **BN**
+- 2022-12-24: Added [-Rei](https://osu.ppy.sh/users/8200707) to **Probationary BN**
+- 2022-12-30: Added [Heroine](https://osu.ppy.sh/users/12727076) to **Probationary BN**
+
```
Comment (aceticke) [inline wiki/People/The_Team/Staff_Log/2022/en.md:620]:
```suggestion
```
nuke this to fix error

**Review by TicClick — APPROVED** [review]

**Pull request merged**